compat/dev_to_tty.c

```diff
@@ -303,6 +303,7 @@ unsigned dev_to_tty(char *restrict ret, unsigned chop, dev_t dev_t_dev, int pid,
   if((flags&ABBREV_PTS) && !strncmp(tmp,"pts/", 4) && tmp[4]) tmp += 4;
   /* gotta check before we chop or we may chop someone else's memory */
   if(chop + (unsigned long)(tmp-buf) <= sizeof buf)
```

Comment:
What if `chop` would be equal `TTY_NAME_SIZE` and `tmp` == `buf`? I think that the problem here is in the `<=`, it should be `<`.